pages.de/linux/aa-complain.md

```diff
@@ -0,0 +1,13 @@
+# aa-complain
+
+> Setze eine AppArmor-Richtlinie in den Beschwerde-Modus.
+> Siehe auch: `aa-disable`, `aa-enforce`, `aa-status`.
+> Weitere Informationen: <https://gitlab.com/apparmor/apparmor/-/wikis/manpage_aa-complain.8>.
+
+- Setze eine Richtlinie in den Beschwerde-Modus:
+
+`sudo aa-complain {{pfad/zum/profil}}`
```

Comment:
According to the translation [template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md).
```suggestion
`sudo aa-complain {{pfad/zu/profil}}`
```

Reply:
Not sure whether the translation template applies here. I think [we have somewhat consistently used „zum“ where grammatically correct](https://github.com/search?q=repo%3Atldr-pages%2Ftldr+path%3A%2F%5Epages%5C.de%5C%2F%2F+pfad%2Fzum&type=code), but some of those are also `pfad/zum/verzeichnis` which the template explicitly forbids.